hack/run-functests.sh

```diff
@@ -1,10 +1,30 @@
 #!/bin/bash
 set -x
-# Install Go 1.20.4
-wget https://go.dev/dl/go1.20.4.linux-amd64.tar.gz
-rm -rf /usr/local/go
-tar -C /usr/local -xzf go1.20.4.linux-amd64.tar.gz
-export PATH=$PATH:/usr/local/go/bin
+
```

Comment:
This part will be reworked as we have a new CI image that has different versions of golang installed. Discussed with @jzding and we will merge this change before adding the updates for the new image